Jasmine/Common/ViewControllers/BaseGameViewControllerDelegate.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 
 /// Sets the shared functionalities that are applicable across all game view controller delegates.
 /// This protocol will be inherited by all the specialsied game view controller delegates.
-protocol BaseGameViewControllerDelegate {
+protocol BaseGameViewControllerDelegate: class {
```

Comment:
I think it is haha. This makes it only be extended to reference types (which are classes). Does not make sense to have a weak reference to value types.